bundles/org.eclipse.osgi.compatibility.state/src/org/eclipse/osgi/internal/resolver/StateImpl.java

```diff
@@ -725,7 +725,6 @@ public StateDelta resolve(BundleDescription[] resolve, boolean discard) {
 	}
 
 	@Override
-	@SuppressWarnings("deprecation")
```

Comment:
The deprecation is actually required @jjohnstn can you have a look? It seems that the method itself does not contain any deprecated call, but the overridden method itself is deprecated!